app/src/androidTest/java/com/sdp/movemeet/view/navigation/security/ActivityDescriptionSecurityTest.java

```diff
@@ -20,16 +26,20 @@
 
     @Before
     public void checkUserNotLogged() {
-        ActivityDescriptionActivity.enableNav = false;
         FirebaseAuth firebaseAuth = FirebaseAuth.getInstance();
         if (firebaseAuth.getCurrentUser() != null) firebaseAuth.signOut();
     }
 
-    //@Rule
-    //public ActivityScenarioRule<ActivityDescriptionActivity> testRule = new ActivityScenarioRule<>(ActivityDescriptionActivity.class);
-
     @Test
     public void redirectionTest() {
-        //onView(withId(R.id.button_login)).check(matches(isDisplayed()));
+        ActivityDescriptionActivity.enableNav = false;
```

Comment:
Why is it better to put this assignment directly inside the test, rather than in the @Before ? Is it for consistency with the name of the method ("checkUserNotLogged"), or is there another reason ?

Reply:
I don't think it's really better or worse. At first I had it in the test just by inattention in my tests but after, I indeed found that compared to the name of the method, it was maybe cleaner to keep this assignment directly in the test.